frontend/packages/noobaa-storage-plugin/src/components/status-card/utils.ts

```diff
@@ -0,0 +1,77 @@
+import * as _ from 'lodash';
+import { HealthState } from '@console/shared/src/components/dashboard/health-card/states';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+import { getGaugeValue } from '../../utils';
+
+export const getObjectStorageState: GetState = (results, error, isLoading) => {
+  const buckets = getGaugeValue(results[0]);
+  const unhealthyBuckets = getGaugeValue(results[1]);
+  const pools = getGaugeValue(results[2]);
+  const unhealthyPools = getGaugeValue(results[3]);
+  const noobaa = results[4];
+  const noobaaPhase = _.get(noobaa, 'data[0].status.phase', null);
```

Comment:
why use `null` loadash returns `undefined` by default?
```suggestion
  const noobaaPhase = _.get(noobaa, 'data[0].status.phase');
```